docs/topics/releases.md

```diff
@@ -66,10 +72,31 @@ plugins {
 </tab>
 </tabs>
 
-If you have projects created with earlier Kotlin versions, change the Kotlin version in your projects and update kotlinx
-libraries if necessary.
+If you have projects created with earlier Kotlin versions, check if you need to also [update the version of any kotlinx
+libraries](gradle-configure-project.md#set-a-dependency-on-a-kotlinx-library).
+
+If you are migrating to a new language release, the Kotlin plugin's migration tools will help you with the migration.
+
+> To learn more about how to work with Gradle in your project, see [Configure a Gradle project](gradle-configure-project.md).
+> 
+{style="tip"}
+
+### Maven
+
+To update to Kotlin %kotlinVersion%, change the version in your `pom.xml` file:
+
+```xml
+<properties>
```

Comment:
I've noticed that our generated by wizard Maven project does not add `<kotlin.version>` in properties.

How about also adding an alternative way here:
or update the plugin version in `<plugins>` block:
```xml
    <build>
        <plugins>
            <plugin>
                <groupId>org.jetbrains.kotlin</groupId>
                <artifactId>kotlin-maven-plugin</artifactId>
                <version>%kotlinVersion%</version>
            </plugin>
        </plugins>
    </build>
```